style.css

```diff
@@ -0,0 +1,79 @@
+
+.title {
+	font-size: 10px;
+}
+
+body{ 
+	font-family:arial; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)